### PR TITLE
Quaker spider

### PR DESCRIPTION
**Pull request by aniversarioperu**
review, not working

### CODE REVIEW

File: onpe_crawler/spiders/quaker.py [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+                            formdata={
+                                '_clase': 'mesas',
+                                '_accion':'displayMesas',
+                                'ubigeo': ubigeo,
+                                'nroMesa': mesa,
+                                'tipoElec': '10',
+                                'page': '1',
+                                'pornumero': '1',
+                            },
+                            meta={'mesa': mesa},
```
Comment (eLRuLL) [inline onpe_crawler/spiders/quaker.py]: missing headers `'X-Requested-With: XMLHttpRequest'`

**Comment by aniversarioperu** [comment]
rebased

**Comment by aniversarioperu** [comment]
squashed

**Comment by aniversarioperu** [comment]
ok, ready for merge

**Pull request merged**